Test/ServerTelemetryChannel.Test/Shared.Tests/Implementation/TransmissionStorageTest.cs

```diff
@@ -654,7 +654,7 @@ public void SkipsTransmissionFileBeingCurrentlyLoadedByAnotherThreadAndTriesLoad
                 Assert.NotNull(dequeued);
             }
 
-            [TestMethod, Timeout(50)]
+            [TestMethod, Timeout(150)]
```

Comment:
I upped the timeout to compensate for the up to 100 ms sleep call in the code under test.